Automattic/jetpack
# Instant Search: Add caching to the api requests

Just a simple 5 minute cache to start out. I was noticing that with starting and stopping filtering on other pages how slow and distracting it can feel to have to do that extra search request even though you just did the same search. Added some comments on some other cases to improve, but decided to try going simple.

bundle size before: 267902 Oct 10 17:54 jp-search.bundle.js
bundle size after: 305240 Oct 10 19:23 jp-search.bundle.js

#### Testing instructions:
* Add define( "JETPACK_SEARCH_PROTOTYPE", true ); to your wp-config.php.
* Ensure that your site has the Jetpack Pro plan and Jetpack Search enabled.
* Add a Jetpack Search widget to the Search page sidebar and configure some filters.
* Start searching and re-searching
